### ISSUE TITLE
additionalimagestore needed as default arg

### ISSUE THREAD

**Issue by lastephey** [NERSC/podman-hpc#87]
We need to include the additional image store as a default argument, otherwise users cannot see their squashed images 

```
stephey@muller:login01:~> podman-hpc images --storage-opt additionalimagestore=$SCRATCH/storage
REPOSITORY                        TAG         IMAGE ID      CREATED       SIZE        R/O
quay.io/podman/hello              latest      e2b3db5d4fdf  3 weeks ago   82.3 kB     true
docker.io/continuumio/miniconda3  latest      a101d1f8cd1c  2 months ago  537 MB      true
docker.io/library/ubuntu          latest      5a81c4b8502e  2 months ago  161 MB      false
stephey@muller:login01:~> podman-hpc images
REPOSITORY                TAG         IMAGE ID      CREATED       SIZE
docker.io/library/ubuntu  latest      5a81c4b8502e  2 months ago  161 MB
```

The only difficulty is that we don't want this on for build, so we'll need to pop this argument out

**Comment by lastephey** [comment]
Actually before I implement the part that pops it out for `build`, @scanon or @danfulton could you please remind me why we didn't want `additionalimagestore` on for build? Or am I remembering wrong? 

I did some brief testing building with it on, and it seems ok.

**Comment by danfulton** [comment]
I thought if you pull and migrate the image, then delete the original and retain the squashed version, and then try to do a build, it will try to use the image from the additionalimage store for your build, and this is problematic.

**Comment by danfulton** [comment]
We might be able to avoid this if we don't copy the hashes over and generate a real hash or use some sort of dummy hash.

**Comment by lastephey** [comment]
Will update to remove it for `pull` and `build`.

**Comment by lastephey** [comment]
Fixed in #88
